opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/measurement_consumer.py

```diff
@@ -32,18 +32,21 @@
 
 class MeasurementConsumer(ABC):
     @abstractmethod
-    def consume_measurement(self, measurement: Measurement) -> None:
+    def consume_measurement(self, measurement: Measurement, **kwargs) -> None:
```

Comment:
This one isn't public yet so I would say we can hold off

Reply:
Ok, removed variadics.